.editorconfig

```
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+root = true
+
+[*]
+end_of_line = lf
```

Since anything at the top level automatically applies to all files within the project there is no need for `[*]`
```suggestion
root = true
end_of_line = lf
```